src/lib/HST/Frontend/HSE/From.hs

```diff
@@ -203,17 +203,16 @@ transformBinds (HSE.IPBinds _ _)    = notSupported "Implicit-parameters"
 -- | Transforms an HSE match into an HST match.
 transformMatch
   :: Member Report r => HSE.Match HSE.SrcSpanInfo -> Sem r (S.Match HSE)
-transformMatch (HSE.Match s name pats rhs mBinds)
-  = S.Match (transformSrcSpan s) <$> transformName name
-  <*> mapM transformPat pats
-  <*> transformRhs rhs
-  <*> mapM transformBinds mBinds
-transformMatch (HSE.InfixMatch s pat name pats rhs mBinds)
-  = S.InfixMatch (transformSrcSpan s) <$> transformPat pat
-  <*> transformName name
-  <*> mapM transformPat pats
-  <*> transformRhs rhs
-  <*> mapM transformBinds mBinds
+transformMatch (HSE.Match s name pats rhs mBinds)          = do
+  S.Match (transformSrcSpan s) False <$> transformName name
+    <*> mapM transformPat pats
+    <*> transformRhs rhs
+    <*> mapM transformBinds mBinds
+transformMatch (HSE.InfixMatch s pat name pats rhs mBinds) = do
```

Comment:
These `do`s should be superfluous:
```suggestion
transformMatch (HSE.Match s name pats rhs mBinds)          =
  S.Match (transformSrcSpan s) False <$> transformName name
    <*> mapM transformPat pats
    <*> transformRhs rhs
    <*> mapM transformBinds mBinds
transformMatch (HSE.InfixMatch s pat name pats rhs mBinds) =
```

Reply:
See da7844918d1b0ca8be3952f73a239aa5039b4aa9.